FluentTerminal.App/Views/SettingsPages/About.xaml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentTerminal.App.ViewModels.Settings;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace FluentTerminal.App.Views.SettingsPages
+{
+    public sealed partial class About : Page
+    {
+        public AboutPageViewModel ViewModel { get; private set; }
+        private string baseUrl = "https://github.com/felixse/FluentTerminal/releases/tag/";
+
+        public bool LatestVersionLoading { get => LatestVersion == null; }
+        public bool LatestVersionFound { get => !LatestVersionNotFound && !LatestVersionLoading; }
+        public bool LatestVersionNotFound { get => LatestVersion == "0.0.0.0"; }
+
+        public string CurrentVersion { get => ViewModel.GetCurrentVersion(); }
+        public string CurrentVersionReleaseNotesURL => baseUrl + CurrentVersion;
+
+        public string LatestVersion { get; private set; }
+        public string LatestVersionReleaseNotesURL => baseUrl + LatestVersion;
+
+        public About()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            if (e.Parameter is AboutPageViewModel viewModel)
+            {
+                ViewModel = viewModel;
+                GetLatestVersion();
+            }
+        }
+
+        private async void GetLatestVersion()
```

Comment:
I figured, but as I mentioned earlier I had trouble getting the page to show the asynchronously loaded latest version. I think it is required to call `Bindings.Update();` once the version is fetched, but this can only be done from the class extending `Page`.

My guess is that the bindings update can be triggered from the ViewModel using an event, but I don't yet understand events in c# good enough to implement this myself...

Alternatively it would be possible to not `await` fetching the latest version, but this freezes the window for the time being so I guess that's not an option 🙃

Reply:
You need to raise NotifyPropertChanged in your ViewModel, that way the View is informed on new values. The base class we use for ViewModels has a method 'Set' which sets the value of the backing field and raises the event. You ca see the exact usage in one of the other ViewModels

Reply:
Should be fixed with db01c51 👍